src/platform/site-wide/announcements/config/index.js

```diff
@@ -1,14 +1,49 @@
+// Node modules.
+import moment from 'moment';
 // Relative imports.
+import Downtime from '../components/Downtime';
 import ExploreVAModal from '../components/ExploreVAModal';
 import FindVABenefitsIntro from '../components/FindVABenefitsIntro';
 import PersonalizationBanner from '../components/PersonalizationBanner';
+import PreDowntime from '../components/PreDowntime';
+import PrePreDowntime from '../components/PrePreDowntime';
 import Profile360Intro from '../components/Profile360Intro';
+import VAMCWelcomeModal, { VAMC_PATHS } from '../components/VAMCWelcomeModal';
 import VAPlusVetsModal from '../components/VAPlusVetsModal';
 import WelcomeToNewVAModal from '../components/WelcomeToNewVAModal';
-import VAMCWelcomeModal, { VAMC_PATHS } from '../components/VAMCWelcomeModal';
+
+// Derive when downtime will start and expire.
+const downtimeStartAtDate = moment('2020-02-29 21:00');
+const downtimeExpiresAtDate = moment('2020-02-29 21:30');
```

Comment:
We need to use moment-timezone, don't we? Since the downtime will start in ET

Reply:
Yupp...

>WARNING: This PR does NOT include support for timezones as using moment-timezone was deemed as bloating the build (for good reason, too). What should I use instead of moment-timezone to support timezones? Any suggestions welcome and greatly appreciated!!

Reply:
Yeah, I think we'll have to bite the bullet on that one, because a solution free of that would probably take a bit